charts/pega/charts/pegasearch/templates/pega-search-deployment.yaml

```diff
@@ -22,6 +22,10 @@ spec:
       labels:
         app: {{ template "searchName" . }}
         component: "Search"
+{{- if .Values.annotations }}
```

Comment:
Let's use `podAnnotations` to better align with the ES charts here: https://github.com/elastic/helm-charts/blob/master/elasticsearch/values.yaml#L49

Reply:
Done

Reply:
looks good